esp/applications/eclwatch/application.yaml

```diff
@@ -0,0 +1,18 @@
+application:
+  name: eclwatch
+  services:
+   - WsSMC
+   - WsWorkunits
+   - WsTopology
+   - WsDfu
+   - WsDfuXRef
+   - WsFileIO
+   - WsPackageProcess
+   - FileSpray
+   - ws_machine
+   - ws_elk
+   - ws_access
+   - ws_account
+   - ws_config
```

Comment:
ws_config is not built in containerized mode.

I'm not convinced it should be part of --application=eclwatch at all, but if it is we need to handle having these files version-dependent or copr with references to services that fail to load.

Reply:
I'll remove it.